functests/utils/profiles/profiles.go

```diff
@@ -3,8 +3,11 @@ package profiles
 import (
 	"context"
 	"fmt"
+	v1 "github.com/openshift/custom-resource-status/conditions/v1"
```

Comment:
please sort imports (yes, it's annoying, I know...)

Reply:
Done.